threema-connector/config/variables.yaml

```diff
@@ -0,0 +1,12 @@
+Variables:
+
+  threema-connector:
+    
+    # Your Threema ID
+    threemaId: ""
+
+    # Secret from your Threema.Gateway Account
+    secret: ""
+    
+    # Private key from your Threema.Gateway Account
+    privateKey: "" 
```

Comment:
we should mark them as '[password]' ... so that these secrets will not be exposed on the engine-cockpit and other frontends that allow inspection of configurations.
https://dev.axonivy.com/doc/10.0/designer-guide/configuration/variables.html